packages/ui-components/src/components/form/Switch.stories.ts

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryObj } from '@storybook/vue3'
+import Switch from './Switch.vue'
+
+export default {
+  component: Switch,
+  parameters: {
+    docs: {
+      description: {
+        component: 'A customizable switch component'
+      }
+    }
+  }
+} as Meta
+
+export const Default: StoryObj = {
+  render: (args) => ({
+    components: { Switch },
+    setup() {
+      return { args }
+    },
+    template: `
+      <div>
+        <Switch 
+          v-bind="args" 
+          @update:modelValue="args.modelValue = $event"
```

Comment:
This is good enough, but in the future I suggest using `ctx.updateArgs()` to update the value (see other stories), cause otherwise the Storybook internal React state won't show that updated value in the Storybook UI (e.g. in the params table)

Reply:
https://github.com/specklesystems/speckle-server/pull/1792/commits/0c37aa146691e9d5b46eac68742c005ac6ed1c64